libraries/AP_Motors/AP_MotorsHeli.h

```diff
@@ -243,6 +249,7 @@ class AP_MotorsHeli : public AP_Motors {
         uint8_t takeoff_collective      : 1;    // true if collective is above 30% between H_COL_MID and H_COL_MAX
         uint8_t below_land_min_coll     : 1;    // true if collective is below H_COL_LAND_MIN
         uint8_t rotor_spooldown_complete : 1;    // true if the rotors have spooled down completely
+        uint8_t start_engine            : 1;
```

Comment:
just for completeness, let's put a comment beside this flag.

Reply:
done. Thanks